template/web/wp-content/themes/<%= slug %>/app/Managers/TaxonomiesManager.php

```diff
@@ -21,9 +22,11 @@ public function run() {
 	/**
 	 * Register custom taxonomies.
 	 *
-	 * @todo use Studiometa\WP_Factory
+	 * Use TaxonomyBuilder from studiometa/wp-toolkit.
+	 *
+	 * @see https://github.com/studiometa/wp-toolkit
+	 *
 	 * @return void
 	 */
-	public function register_taxonomies() {
-	}
+	public function register_taxonomies() {}
```

Comment:
**question:** Same as for the CPT manager, should we rename this method to `register_taxonomies()`?